### PR TITLE
Separate stage for windows exe installer and build tarballs on Windows

### PR DESCRIPTION
**Pull request by musm**
Two changes:
1. Build tarballs on Windows platforms, as is done for the other supported platforms.
2. Separate out the windows  exe installer into a new stage

### CODE REVIEW

**Comment by musm** [comment]
Needs https://github.com/JuliaCI/julia-buildbot/pull/146/files to pass on windows

**Comment by musm** [comment]
g2g @staticfloat?

**Comment by staticfloat** [comment]
Yep, lgtm

File: Makefile [hunk 1]
```diff
@@ -465,16 +465,15 @@ endif
 ifeq ($(OS), Darwin)
 	-cat $(JULIAHOME)/contrib/mac/startup.jl >> $(DESTDIR)$(prefix)/etc/julia/startup.jl
 endif
-
 ifeq ($(OS), WINNT)
 	cd $(BUILDROOT)/julia-$(JULIA_COMMIT)/bin && rm -f llvm* llc.exe lli.exe opt.exe LTO.dll bugpoint.exe macho-dump.exe
+endif
+	cd $(BUILDROOT) && $(TAR) zcvf $(JULIA_BINARYDIST_FILENAME).tar.gz julia-$(JULIA_COMMIT)
 
+exe:
```
Comment (musm) [inline Makefile]: Looks like I forgot `exe: binary-dist`

Comment (musm) [reply Makefile]: Do we really want to slow the build process down by adding this? FWIW copying all the files back on my computer takes 2-5 minutes

Comment (staticfloat) [reply Makefile]: Yes, we do.  It's better to haev a slow build to create something we know is correct, then for there to be a possibility to have old files shipped.

**Pull request merged**